specs/while_loop.yaml

```diff
@@ -1,14 +0,0 @@
-name: "While loop"
-tags: ["shell"]
-command: "while {{condition}}; do\n\t{{command}}\ndone"
-arguments:
-  -
-    name: "condition"
-    description: "The condition for the while loop"
-  -
-    name: "command"
```

Comment:
This was a repeat of another one

Reply:
thoughts on deleting all of the top level workflows on top of just this one? I feel like they're already encapsulated within other workflows anyway

Reply:
yeah I think that's a good idea